docs/guides/first-shopify-abtest.mdx

```diff
@@ -0,0 +1,140 @@
+
+![image](https://user-images.githubusercontent.com/1315028/150182063-fdc2bc10-48e1-414f-a05a-dd32c55a376f.png)
+
+- Click on the **Customize** button 
+
+![image](https://user-images.githubusercontent.com/1315028/150182338-065de17e-a33d-4141-9b34-9518a53b60eb.png)
+
+- From the left hand navigation menu, click on **Image banner**
+
+![image](https://user-images.githubusercontent.com/1315028/150182526-dd77fea2-20b9-4812-b6f9-30ab309a6a49.png)
+
+- Under the **First Image**, click **Select Image** button 
+
+![image](https://user-images.githubusercontent.com/1315028/150182728-2cae4794-0a08-4781-9d5f-6cbfe7394d48.png)
+
+- Select the hero banner image that you want to use as control from the library or upload a new image
+
+![image](https://user-images.githubusercontent.com/1315028/150182819-5a6e1745-7e54-4d02-a3e4-406eb278da8d.png)
+
+- Repeat the previous two steps for the **Second Image** as well
+- At this point, you should see both images in your hero banner on the store 
+
+
+### Step 2: Create an experiment on Statsig
+
+To create an experiment on Statsig, 
+- Log into https://console.statsig.com/
+- Click on **Experiments** from the left hand navigation menu
+- Click on **Create**, then enter the name and description of the experiment
+- In the experiment **Setup** tab, enter the names of the variants that you want to test in the **Groups** panel; in this case I'm just calling them Control and Test. 
+- Staying in the **Groups** panel, click on **sample_parameter** and enter a parameter name of your choice; in this case, I'm calling the parameter **image_selector**. Enter the parameter values that you want to use; in this case, I simply want to toggle between the two images that we've already uploaded to the store in the previous step
+
+![image](https://user-images.githubusercontent.com/1315028/150181235-068a1cf3-00ef-48f8-b2fa-51c0338e5e6d.png)
+
+
+### Step 3: Customize your theme
+
+After you log into your store in step 1, click on **Themes** under the **Online Store** in the left hand navigation panel. 
+Then, 
+- Click on the **Actions** drop down, then click on **Edit Code**
+- Click on **theme.liquid** file under the **Layout** section
+- Load the SDK by copying the code the below in the `head` section; use the experiment config to show the right image to your visitors
+
+```js
+<script src="https://cdn.jsdelivr.net/npm/statsig-js/build/statsig-prod-web-sdk.min.js"></script>
+```
+
+
+- Initialize the SDK and get the experiment config by copying the following in the `body` section
+
+```js
+<script>
+    // hide the page until we fetch the configuration from statsig
+    document.getElementsByTagName("BODY")[0].style.display = "none";
+    
+    //initialize the Statsig SDK using your Statsig client SDK key 
+    window.statsig.initialize("<CLIENT_SDK_KEY>").then(() => {
+        // Assuming you're on the Dawn theme and you've uploaded two images for the hero image
```

Comment:
people might glance over this but they need to put in an SDK key and an experiment name further down